notebooks/databricks.py

```diff
@@ -37,30 +33,26 @@
 
 # COMMAND ----------
 
-p.run("transform")
-
-# COMMAND ----------
-
-p.run("train")
+training_data = p.get_artifact("training_data")
+training_data.describe()
```

Comment:
Nit: We should replicate this in the example as well. 
It is a much better experience to see this data before running all the other steps

Reply:
SG. I'll do another update in the example repo.